src/Tools/CommonNetCoreReferences/project.json

```diff
@@ -1,52 +1,52 @@
-    "System.Text.Encoding.Extensions": "4.0.11-rc2-23931",
-    "System.Threading": "4.0.11-rc2-23931",
-    "System.Threading.Tasks": "4.0.11-rc2-23931",
-    "System.Threading.Tasks.Parallel": "4.0.1-rc2-23931",
-    "System.Threading.Thread": "4.0.0-rc2-23931",
-    "System.Xml.ReaderWriter": "4.0.11-rc2-23931",
-    "System.Xml.XDocument": "4.0.11-rc2-23931",
-    "System.Xml.XmlDocument": "4.0.1-rc2-23931",
-    "System.Xml.XPath.XDocument": "4.0.1-rc2-23931"
+    "Microsoft.NETCore.Portable.Compatibility": "1.0.1-rc2-24018",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc2-24018",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc2-24018",
+    "Microsoft.NETCore.TestHost": "1.0.0-rc2-24018",
+    "System.AppContext": "4.1.0-rc2-24018",
+    "System.Collections": "4.0.11-rc2-24018",
+    "System.Collections.Concurrent": "4.0.12-rc2-24018",
+    "System.Collections.Immutable": "1.2.0-rc2-24018",
+    "System.Console": "4.0.0-rc2-24018",
+    "System.Diagnostics.Debug": "4.0.11-rc2-24018",
+    "System.Diagnostics.FileVersionInfo": "4.0.0-rc2-24018",
+    "System.Diagnostics.Process": "4.1.0-rc2-24018",
+    "System.Diagnostics.StackTrace": "4.0.1-rc2-24018",
+    "System.Diagnostics.Tools": "4.0.1-rc2-24018",
+    "System.Dynamic.Runtime": "4.0.11-rc2-24018",
+    "System.Globalization": "4.0.11-rc2-24018",
+    "System.IO.FileSystem": "4.0.1-rc2-24018",
+    "System.IO.FileSystem.Primitives": "4.0.1-rc2-24018",
+    "System.IO.FileSystem.Watcher": "4.0.0-rc2-24018",
+    "System.IO.Pipes": "4.0.0-rc2-24018",
+    "System.Linq": "4.1.0-rc2-24018",
+    "System.Linq.Expressions": "4.0.11-rc2-24018",
+    "System.Net.NameResolution": "4.0.0-rc2-24018",
+    "System.Net.Sockets": "4.1.0-rc2-24018",
+    "System.Reflection": "4.1.0-rc2-24018",
+    "System.Reflection.Primitives": "4.0.1-rc2-24018",
+    "System.Resources.ResourceManager": "4.0.1-rc2-24018",
+    "System.Runtime": "4.1.0-rc2-24018",
+    "System.Runtime.Extensions": "4.1.0-rc2-24018",
+    "System.Runtime.Handles": "4.0.1-rc2-24018",
+    "System.Runtime.InteropServices": "4.1.0-rc2-24018",
+    "System.Runtime.Loader": "4.0.0-rc2-24018",
+    "System.Runtime.Numerics": "4.0.1-rc2-24018",
+    "System.Security.Cryptography.Algorithms": "4.1.0-rc2-24018",
+    "System.Security.Cryptography.Encoding": "4.0.0-rc2-24018",
+    "System.Security.Cryptography.X509Certificates": "4.1.0-rc2-24018",
+    "System.Text.Encoding": "4.0.11-rc2-24018",
+    "System.Text.Encoding.CodePages": "4.0.1-rc2-24018",
+    "System.Text.Encoding.Extensions": "4.0.11-rc2-24018",
+    "System.Threading": "4.0.11-rc2-24018",
+    "System.Threading.Tasks": "4.0.11-rc2-24018",
+    "System.Threading.Tasks.Parallel": "4.0.1-rc2-24018",
+    "System.Threading.Thread": "4.0.0-rc2-24018",
+    "System.Xml.ReaderWriter": "4.0.11-rc2-24018",
+    "System.Xml.XDocument": "4.0.11-rc2-24018",
+    "System.Xml.XmlDocument": "4.0.1-rc2-24018",
+    "System.Xml.XPath.XDocument": "4.0.1-rc2-24018"
```

Comment:
At this point, what is stopping us from referencing the metapackage instead of all of these individually?